Sources/AppStoreConnectCLI/Services/Operations/ListBetaTestersByGroupOperation.swift

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBetaTestersByGroupOperation: APIOperation {
+
+    struct Options {
+        let groupId: String
+    }
+
+    enum Error: LocalizedError {
+        case notFound
+
+        var errorDescription: String? {
+            switch self {
+            case .notFound:
+                return "Beta testers not found."
+            }
+        }
+    }
```

Comment:
I think It's ok for a group to have 0 testers. We are not throwing not found error in other listing operations as well.
```suggestion
```